easybuild/easyconfigs/r/ROOT/ROOT-6.26.06-foss-2022a.eb

```diff
@@ -0,0 +1,55 @@
+name = 'ROOT'
+version = '6.26.06'
+
+homepage = 'https://root.cern.ch/drupal/'
+description = """The ROOT system provides a set of OO frameworks with all the functionality
+    needed to handle and analyze large amounts of data in a very efficient way."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://root.cern.ch/download/']
+sources = ['%(namelower)s_v%(version)s.source.tar.gz']
+checksums = ['b1f73c976a580a5c56c8c8a0152582a1dfc560b4dd80e1b7545237b65e6c89cb']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('GSL', '2.7'),
+    ('libxml2', '2.9.13'),
+    ('PCRE', '8.45'),
+    ('CFITSIO', '4.1.0'),
+    ('freetype', '2.12.1'),
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('zlib', '1.2.12'),
+    ('X11', '20220504'),
+    ('Mesa', '22.0.3'),
+    ('libGLU', '9.0.2'),
+    ('GL2PS', '1.4.2'),
+    ('FFTW', '3.3.10'),
+    ('SQLite', '3.38.3'),
+    ('XZ', '5.2.5'),
+    ('libpng', '1.6.37'),
+]
+
+# NOTE: Ensure that each configopts string begins with a blank
+# disable some components
+configopts = " -Dxrootd=OFF -Dmysql=OFF -Dkrb5=OFF -Dodbc=OFF -Doracle=OFF -Dpgsql=OFF -Dqt=OFF"
+
+# make sure some components are enabled
+configopts += " -Dpcre=ON -Dzlib=ON -Dpyroot=ON"
+configopts += " -Dunuran=ON -Dexplicitlink=ON -Dminuit2=ON -Droofit=ON "
+
+# Add component-specific settings based on dependencies
+configopts += ' -Dfftw3=ON -Dgsl=ON -DOpenGL_GL_PREFERENCE=GLVND'
+
+# Set C++ standard to C++17 for better stability
+configopts += ' -DCMAKE_CXX_STANDARD=17'
+
+configopts += " -Dgallium-drivers=swrast"
```

Comment:
isn't this an build option for Mesa? I'm surprised to see it here

Reply:
Yeah, it looks like that's a mistake, it slipped in via #14862, and since CMake doesn't seem to complain about it, it stayed there...

So, I'll strip it out, and open a PR to do the same in existing `ROOT` easyconfigs where it's used.